torch/csrc/lazy/backend/backend_interface.cpp

```diff
@@ -0,0 +1,41 @@
+#include <torch/csrc/lazy/backend/backend_interface.h>
+
+namespace torch {
+namespace lazy {
+
+std::atomic<const BackendImplInterface*> backend_impl_registry;
```

Comment:
Oops, you might need to rebase after I land: https://github.com/pytorch/pytorch/pull/69064.

Reply:
Yes, I noticed that. I will merge after you land yours. I also realized I need to add TORCH_API in those header files.

Reply:
Yeah, most of those are not inline.